src/AutoAffix.js

```diff
@@ -32,6 +32,11 @@ class AutoAffix extends React.Component {
     this._windowScrollListener = addEventListener(
       ownerWindow(this), 'scroll', () => this.onWindowScroll()
     );
+
+    this._windowResizeListener = addEventListener(
```

Comment:
yeah I figured since we aren't getting every case it couldn't hurt to catch a few more cases. makes more sense with autoWidth